src/js/sdk.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  defaultIntegrations,
+  getCurrentHub,
+  Integrations
+} from "@sentry/browser";
+import { RewriteFrames } from "@sentry/integrations";
+import { initAndBind } from "@sentry/core";
+import { setExtra } from "@sentry/minimal";
+import { StackFrame } from "@sentry/types";
+
+import { ReactNativeOptions } from "./backend";
+import { ReactNativeClient } from "./client";
+import {
+  DeviceContext,
+  ReactNativeErrorHandlers,
+  Release
+} from "./integrations";
+
+/**
+ * Inits the SDK
+ */
+export function init(
+  options: ReactNativeOptions = {
+    enableNative: true,
+    enableNativeCrashHandling: true
+  }
+): void {
+  if (options.defaultIntegrations === undefined) {
+    options.defaultIntegrations = [
+      new ReactNativeErrorHandlers(),
+      new Release(),
+      ...defaultIntegrations.filter(
+        integration =>
+          integration.name !== "GlobalHandlers" && // We will use the react-native internal handlers
```

Comment:
I'd change that slightly to say

```js
...defaultIntegrations.filter(i => !IGNORED_DEFAULT_INTEGRATIONS.includes(i))
```

and somewhere at the top

```js
const IGNORED_DEFAULT_INTEGRATIONS = [
"GlobalHandlers" && // We will use the react-native internal handlers
 

 
I'd change that slightly to say

```js
...defaultIntegrations.filter(i => !IGNORED_DEFAULT_INTEGRATIONS.includes(i))
```

and somewhere at the top

```js
const IGNORED_DEFAULT_INTEGRATIONS = [
  "GlobalHandlers", // We will use the react-native internal handlers
  "Breadcrumbs", // We add it later, just not patching fetch
  "TryCatch" // We don't need this
]
```